packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/MrwSerializationTypeDefinitions/ModelCustomizationTests.cs

```diff
@@ -174,5 +174,32 @@ public async Task CanCustomizeNullableStringToFixedEnum()
             var file = writer.Write();
             Assert.AreEqual(Helpers.GetExpectedFromFile(), file.Content);
         }
+
+        [Test]
+        public async Task CanCustomizeEnumToFrameworkType()
```

Comment:
nit: could we also add a test that tests the same thing but with the property customized to a field?